cli/src/lib.rs

```diff
@@ -27,16 +31,25 @@ pub fn run() {
         let path = path.as_path().to_str().expect("wrong path");
 
         println!("[sled-storage] connected to {}", path);
-        run(SledStorage::new(path).expect("failed to load sled-storage"));
+        run(
+            SledStorage::new(path).expect("failed to load sled-storage"),
+            args.execute,
+        );
     } else {
         println!("[memory-storage] initialized");
-        run(MemoryStorage::default());
+        run(MemoryStorage::default(), args.execute);
     }
 
-    fn run<T, U: GStore<T> + GStoreMut<T>>(storage: U) {
+    fn run<T, U: GStore<T> + GStoreMut<T>>(storage: U, input: Option<PathBuf>) {
         let output = std::io::stdout();
         let mut cli = Cli::new(storage, output);
 
+        if let Some(path) = input {
+            if let Err(e) = cli.load(path.as_path().to_str().expect("wrong path")) {
+                println!("[error] {}\n", e);
```

Comment:
[nit]
Right below code prints meesage using eprintln! macro. So how about using eprintln! here also?

Reply:
It's not std's println! that's used here.

I thought about this too, but noticed that there is a separate macro declaration above.

Why not change the name in another PR? @panarch

Reply:
Interesting.. I think that we might not need to use `stderr` pipe at all in here.
In `lib.rs`,
https://github.com/gluesql/gluesql/blob/69211ebe96158e0ecf1d8fc373dbf0596899ad49/cli/src/lib.rs#L40-L42

This is the only case we use `eprintln!`. All else, we use `stdout`.
Considering this is CLI for interacting with GlueSQL, it would be better simply not to use `stderr`.
Then, we can think this as a different task and do it in the different PR.

How do you think? @24seconds @ever0de

Reply:
Looks good

Reply:
looks good